### PR TITLE
[fix] angularjs fix tests, update npm

### PR DESCRIPTION
**Pull request by a13ks3y**
[fix] https://github.com/tastejs/todomvc/issues/1616

### CODE REVIEW

**Comment by addyosmani** [comment]
We're currently failing with the following warning in CI:

```
─ xmlbuilder@8.2.2 
/home/travis/build/tastejs/todomvc/tests/node_modules/drool/node_modules/selenium-webdriver/lib/promise.js:654
    throw error;
    ^
NoSuchElementError: no such element: Unable to locate element: {"method":"css selector","selector":".todo-list li"}
  (Session info: chrome=54.0.2815.0)
  (Driver info: chromedriver=2.22.397932 (282ed7cf89cf0053b6542e0d0f039d4123bbb6ad),platform=Linux 3.13.0-40-generic x86_64)
    at WebDriverError (/home/travis/build/tastejs/todomvc/tests/node_modules/drool/node_modules/selenium-webdriver/lib/error.js:27:5)
    at NoSuchElementError (/home/travis/build/tastejs/todomvc/tests/node_modules/drool/node_modules/selenium-webdriver/lib/error.js:242:5)
```

**Comment by addyosmani** [comment]
Friendly ping @a13ks3y

**Comment by a13ks3y** [comment]
@addyosmani will check it later, probably this pull request is not actual anymore.

File: examples/angularjs/node_modules/angular/angular.js [hunk 1]
```diff
@@ -1,9 +1,9 @@
 /**
- * @license AngularJS v1.4.3
- * (c) 2010-2015 Google, Inc. http://angularjs.org
+ * @license AngularJS v1.5.5
```
Comment (kveretennicov) [inline examples/angularjs/node_modules/angular/angular.js]: I suspect this is what broke the CI tests. There is a backward-incompatible change in AngularJS 1.5.x, which breaks this example. See angular/angular.js#15642.

**Comment by FadySamirSadek** [comment]
@a13ks3y The test suite has been migrated to cypress which should solve the selenium driver issue can you please rebase your branch and handle merge 2 conflicting files ?

**Pull request closed without merging**